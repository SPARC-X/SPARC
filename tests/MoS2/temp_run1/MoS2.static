Total free energy (Ha): -8.908400707847865E+01
Atomic forces (Ha/Bohr):
 -2.3517689069E-02   3.8929296489E-02   1.0727512162E-01
  1.2526880851E-02  -1.1082132242E-02  -6.8717106161E-02
  1.0990808219E-02  -2.7847164247E-02  -3.8558015461E-02
Stress (Ha/Bohr**2): 
  9.6382887010E-05   3.7463828085E-05 
  3.7463828085E-05   1.6593752286E-03 
Stress equiv. to all periodic (GPa): 
  9.4522747720E-02   3.6740795805E-02 
  3.6740795805E-02   1.6273501549E+00 
