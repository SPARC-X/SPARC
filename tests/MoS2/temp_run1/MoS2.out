***************************************************************************
*                       SPARC (version Apr 23, 2021)                      *
*   Copyright (c) 2020 Material Physics & Mechanics Group, Georgia Tech   *
*           Distributed under GNU General Public License 3 (GPL)          *
*                   Start time: Mon May 10 20:57:53 2021                  *
***************************************************************************
                           Input parameters                                
***************************************************************************
CELL: 6.1 6.1 30 
LATVEC:
1 0 0 
-0.5 0.8666 0 
0 0 1 
FD_GRID: 51 51 250
FD_ORDER: 12
BC: P P D
KPOINT_GRID: 3 3 1
KPOINT_SHIFT: 0 0 0
SPIN_TYP: 0
ELEC_TEMP_TYPE: Fermi-Dirac
SMEARING: 0.0009999935878
CHEB_DEGREE: 49
NSTATES: 20
EXCHANGE_CORRELATION: GGA_PBE
CALC_STRESS: 1
CHEFSI_BOUND_FLAG: 0
MAXIT_SCF: 100
MINIT_SCF: 3
MAXIT_POISSON: 3000
TOL_SCF: 1.00E-06
POISSON_SOLVER: AAR
TOL_POISSON: 1.00E-08
TOL_LANCZOS: 1.00E-02
TOL_PSEUDOCHARGE: 1.00E-08
MIXING_VARIABLE: potential
MIXING_PRECOND: none
MIXING_PARAMETER: 0.3
MIXING_HISTORY: 7
PULAY_FREQUENCY: 1
PULAY_RESTART: 0
REFERENCE_CUTOFF: 0.5
RHO_TRIGGER: 4
FIX_RAND: 0
VERBOSITY: 1
PRINT_FORCES: 1
PRINT_ATOMS: 0
PRINT_EIGEN: 0
PRINT_DENSITY: 0
OUTPUT_FILE: MoS2
***************************************************************************
                           Parallelization                                 
***************************************************************************
NP_SPIN_PARAL: 1
NP_KPOINT_PARAL: 5
NP_BAND_PARAL: 1
NP_DOMAIN_PARAL: 1 1 9
NP_DOMAIN_PHI_PARAL: 2 2 12
EIG_SERIAL_MAXNS: 2000
***************************************************************************
                             Initialization                                
***************************************************************************
Number of processors               :  48
Mesh spacing in x-direction        :  0.119608 (Bohr)
Mesh spacing in y-direction        :  0.119608 (Bohr)
Mesh spacing in z direction        :  0.12 (Bohr)
Number of symmetry adapted k-points:  5
Output printed to                  :  MoS2.out
Total number of atom types         :  2
Total number of atoms              :  3
Total number of electrons          :  26
Atom type 1  (valence electrons)   :  Mo 14
Pseudopotential                    :  Mo.psp8
Pseudocharge radii of atom type 1  :  6.70 6.70 6.72 (x, y, z dir)
Number of atoms of type 1          :  1
Atom type 2  (valence electrons)   :  S 6
Pseudopotential                    :  S.psp8
Pseudocharge radii of atom type 2  :  6.70 6.70 6.72 (x, y, z dir)
Number of atoms of type 2          :  2
Estimated total memory usage       :  5.97 GB
Estimated memory per processor     :  127.43 MB
===================================================================
                    Self Consistent Field (SCF#1)                     
===================================================================
Iteration     Free Energy (Ha/atom)   SCF Error        Timing (sec)
1            -2.9585211539E+01        1.186E-01        62.771
2            -2.9670965169E+01        1.776E-01        16.841
3            -2.9687642716E+01        1.405E-01        16.861
4            -2.9693916986E+01        4.352E-02        16.629
5            -2.9693924824E+01        4.884E-02        16.370
6            -2.9694450878E+01        1.581E-02        16.558
7            -2.9694627977E+01        1.161E-02        16.184
8            -2.9694667080E+01        2.795E-03        16.176
9            -2.9694668356E+01        1.515E-03        16.010
10           -2.9694668790E+01        6.373E-04        15.791
11           -2.9694669180E+01        5.117E-05        15.745
12           -2.9694669200E+01        2.214E-05        15.476
13           -2.9694669089E+01        2.405E-05        15.247
14           -2.9694669047E+01        6.413E-06        15.266
15           -2.9694668848E+01        6.060E-06        15.015
16           -2.9694668973E+01        2.887E-06        15.196
17           -2.9694669083E+01        2.121E-06        14.918
18           -2.9694669134E+01        1.306E-06        14.987
19           -2.9694669047E+01        1.346E-06        15.002
20           -2.9694669026E+01        4.751E-07        15.251
Total number of SCF: 20    

k = [-0.333, -0.333, 0.000]
Occupation of state 18 = 0.000000000000000.
Occupation of state 20 = 0.000000000000000.
====================================================================
                    Energy and force calculation                    
====================================================================
Free energy per atom               : -2.9694669026E+01 (Ha/atom)
Total free energy                  : -8.9084007078E+01 (Ha)
Band structure energy              : -2.1030757160E+01 (Ha)
Exchange correlation energy        : -1.2270360074E+01 (Ha)
Self and correction energy         : -2.1198465860E+02 (Ha)
Entropy*kb*T                       : -3.3304207072E-07 (Ha)
Fermi level                        : -1.7627243674E-01 (Ha)
RMS force                          :  7.8685796476E-02 (Ha/Bohr)
Maximum force                      :  1.1651833994E-01 (Ha/Bohr)
Time for force calculation         :  0.203 (sec)
Maximum stress                     :  1.6593752286E-03 (a.u.)
Time for stress calculation        :  0.447 (sec)
***************************************************************************
                               Timing info                                 
***************************************************************************
Total walltime                     :  365.284 sec
___________________________________________________________________________

***************************************************************************
*             Material Physics & Mechanics Group, Georgia Tech            *
*                       PI: Phanish Suryanarayana                         *
*                Main Developers: Qimen Xu, Abhiraj Sharma                *
*     Collaborators: J.E. Pask (LLNL), A.J. Medford (GT), E. Chow (GT)    *
*  Acknowledgements: U.S. DOE (DE-SC0019410); U.S. NSF (1333500,1553212)  *
***************************************************************************
                                                                           
